src/base/static/js/views/app-view.js

```diff
@@ -221,6 +224,23 @@ module.exports = Backbone.View.extend({
       router: this.options.router,
     }).render();
 
+    // TODO(jalmogo): refactor this into our Header component, when ready:
+    const datasetSlug = storeState.config.app.dataset_download.slug;
+    const datasetOwner = storeState.config.app.dataset_download.owner;
+    const apiRoot = storeState.config.app.api_root;
+    const isAdmin = Util.getAdminStatus(datasetSlug);
+    if (isAdmin) {
+      ReactDOM.render(
+        <Link
+          href={`${apiRoot}${datasetOwner}/datasets/${datasetSlug}/mapseed-places.csv?format=csv&include_private=true&page_size=10000`}
+          style={{ float: "right" }}
+        >
+          <SecondaryButton>{`Download Survey Data`}</SecondaryButton>
```

Comment:
I don't think the nested `SecondaryButton` is needed here-- I think the `Link` component by itself should be enough.

Reply:
I suppose a link looks a bit cleaner. This is how it currently looks with just the link:
![download-data-header](https://user-images.githubusercontent.com/42042460/44185216-d78bce80-a0c7-11e8-8802-136af6981970.png)

in re:
>An alternative UI might nest the download button in the flyout that appears when you click on a logged-in user avatar. 

I think that is preferable. Downloading the dataset is an infrequently used feature, so it only clutters the limited mobile space by keeping it up there.

I may need to port the PagesNavView view to React, so I'll explore whether that is worth doing.